spacy/tests/doc/test_span.py

```diff
@@ -34,7 +34,6 @@ def doc(en_tokenizer):
         lemmas=lemmas,
     )
 
-
```

Comment:
We format all files in the main repo with `black` - you can find more information [here](https://github.com/explosion/spaCy/blob/master/CONTRIBUTING.md#code-conventions). If you run `black` you'll see that this newline will be added in again ;-)